analyzer/src/main/resources/scripts/init.gradle

```diff
@@ -167,6 +172,25 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
                         [], [], [error], [])
             }
 
+            project.gradle.getIncludedBuilds().each {
+                project.logger.warn("Processing included build '$it.name, $it.projectDir'.")
+                // Since the projects of an included build are not present in the public API, the internal API is used
+                // here with fully qualified name to minimize error when unsupported versions of Gradle are used.
+                def state = (it as org.gradle.internal.composite.IncludedBuildInternal).getTarget()
+                if (state as org.gradle.internal.build.IncludedBuildState) {
+                    state.ensureProjectsConfigured()
+                    def build = state.getMutableModel()
+                    def includedRootProject = build.rootProject.project
+                    String identifier = toIdentifier(
+                            includedRootProject.group.toString(),
+                            includedRootProject.name,
+                            includedRootProject.version.toString()
+                    )
+                    project.logger.warn("Mapping included build '$it.name' to root project '$identifier'.")
```

Comment:
Consider `quiet` instead of `warn`.

Reply:
`error`, `quiet` and `warn` are all enabled by default, see: https://docs.gradle.org/current/userguide/logging.html